pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1912,6 +1918,9 @@ self: super: {
   # https://github.com/Gabriella439/Haskell-Pipes-HTTP-Library/pull/17
   pipes-http = doJailbreak super.pipes-http;
 
+  # 2024-09-18: transformers <0.6
+  pipes-extras = doJailbreak super.pipes-extras;
```

Comment:
Why is this necessary? It was fixed in revision 7: https://hackage.haskell.org/package/pipes-extras-1.0.15/revisions/

Reply:
I couldn't figure out why! In `hackage-packages.nix`, pipes-extras has `revision = "8"`. In any case, adding a version assertion is a good idea.

Reply:
Aha. The test-suite component still has `transformers <0.6`.

https://hackage.haskell.org/package/pipes-extras-1.0.15/revision/8.cabal